packages/core/package.json

```diff
@@ -39,6 +39,7 @@
         "@types/react-dom": "^18.0.0",
         "babel-loader": "^8.1.0",
         "circular-dependency-plugin": "^5.2.2",
+        "@testing-library/user-event": "^13.2.1",
```

Comment:
i dont think that we need to add this here, if we have it in the root package.json
like for instance testing-library/react which is not listed here

Reply:
Yes @ali-hosseini-fs you are right, will remove this